### ISSUE TITLE
finish, test and put the packages on the website

### ISSUE THREAD

**Issue by LostNeophyte** [LunarVim/LunarVim-mono#3]

**Comment by Burnsedia** [comment]
Need help with this?

**Comment by LostNeophyte** [comment]
yes, help would be very much appreciated, the packages are probably finished and all working but I didn't test it so it would be very nice if someone tested them, I'm on fedora so I can test the appimage, maybe .deb in docker and if you're not on windows the .exe (this one is painful) if you're on mac, could you check if `lvim-macos.tar.gz` works for you?

before testing any of them move your `.local/share/lunarvim`, `.cache/lvim`, `.local/state/lvim` folders and back them up somewhere, so the packages start on a clean system

**Comment by Burnsedia** [comment]
I have a windows VM, I am making a MacOS VM, and I have an old mac somewhere.

**Comment by Burnsedia** [comment]
Just messaged you on Discord, I am not a big discord user
